docs/scripts_md/imaging_upload_file.md

```diff
@@ -0,0 +1,93 @@
+# NAME
+
+imaging\_upload\_file.pl -- a single step script for the imaging pre-processing
+and insertion pipeline sequence
+
+# SYNOPSIS
+
+perl imaging\_upload\_file.pl &lt;/path/to/UploadedFile> \`\[options\]\`
+
+Available options are:
+
+\-profile      : name of the config file in
+                `../dicom-archive/.loris_mri`
+
+\-upload\_id    : The upload ID of the given scan uploaded
+
+\-verbose      : if set, be verbose
+
+# DESCRIPTION
+
+The program does the following
+
+\- Gets the location of the uploaded file (.zip,.tar.gz or .tgz)
+
+\- Unzips the uploaded file
+
+\- Sources the Environment
+
+\- Uses the ImagingUpload class to :
+   1) Validate the uploaded file   (set the validation to true)
+   2) Run dicomtar.pl on the file  (set the dicomtar to true)
+   3) Run tarchiveLoader on the file (set the minc-created to true)
+   4) Removes the uploaded file once the previous steps have completed
+   5) Update the mri\_upload table
+
+## Methods
+
+### getPnameUsingUploadID()
```

Comment:
To be consistent with all other documentation, include the inputs in the function name.
Example: getPnameUsingUploadID($upload_id)
  
Valid for all functions presented in the script